tests/queries/0_stateless/02041_openssl_hash_functions_test.reference

```diff
@@ -0,0 +1,8 @@
+098F6BCD4621D373
+DB346D691D7ACC4DC2625DB19F9E3F52
+098F6BCD4621D373CADE4E832627B4F6
+A94A8FE5CCB19BA61C4C0873D391E987982FBBD3
+90A3ED9E32B2AAF4C61C410EB925426119E1A9DC53D4286ADE99A809
+9F86D081884C7D659A2FEAA0C55AD015A3BF4F1B2B0B822CD15D6C15B0F00A08
+768412320F7B0AA5812FCE428DC4706B3CAE50E02A64CAA16A782249BFE8EFC4B7EF1CCB126255D196047DFEDF17A0A9
+EE26B0DD4AF7E749AA1A8EE3C10AE9923F618980772E473F8819A5D4940E0DB27AC185F8A0E1D5F84F88BC887FD67B143732C304CC5FA9AD8E6F57F50028A8FF
```

Comment:
Reference file can be generated easily with
`clickhouse-client -n < file.sql > file.reference`

(now you have missing newline)

Reply:
Fixed.